openslides_backend/action/action_worker.py

```diff
@@ -0,0 +1,288 @@
+    action_worker_thread.start()
+    while not action_worker_thread.started:
+        sleep(0.001)  # The action_worker_thread should gain the lock and NOT this one
+    if lock.acquire(timeout=THREAD_WATCH_TIMEOUT):
+        if hasattr(action_worker_thread, "exception"):
+            logger.debug("Action request finished with exception within timeout.")
+            lock.release()
+            raise action_worker_thread.exception
+        if hasattr(action_worker_thread, "response"):
+            logger.debug("Action request finished successfully within timeout.")
+            lock.release()
+            return action_worker_thread.response
+        lock.release()
+        raise ActionException(
+            "Action request ended with unknown reason, probably an unexpected timeout!"
+        )
+
+    message = action_worker_writing.initial_action_worker_write()
+    return ActionsResponse(
+        status_code=HTTPStatus.ACCEPTED.value,
+        success=False,
+        message=message,
+        results=[
+            [
+                {
+                    "fqid": action_worker_writing.fqid,
+                    "name": action_worker_writing.action_names,
+                    "written": action_worker_writing.written,
+                }
+            ]
+        ],
+    )
+
+
+class ActionWorkerWriting(object):
+    def __init__(
+        self,
+        user_id: int,
+        starttime: int,
+        logging: LoggingModule,
+        action_names: str,
+        datastore: DatastoreService,
+    ) -> None:
+        self.user_id = user_id
+        self.starttime = starttime
+        self.logger = logging.getLogger(__name__)
+        self.action_names = action_names
+        self.datastore = datastore
+
+        self.new_id: Optional[int] = None
+        self.fqid: str = "Still not set"
+        self.written: bool = False
+
+    def initial_action_worker_write(self) -> str:
+        current_time = round(time())
+        with self.datastore.get_database_context():
+            if not self.new_id:
+                self.new_id = self.datastore.reserve_id(collection="action_worker")
+                self.fqid = fqid_from_collection_and_id("action_worker", self.new_id)
+            try:
+                self.datastore.write_action_worker(
+                    WriteRequest(
+                        events=[
+                            Event(
+                                type=EventType.Create,
+                                fqid=self.fqid,
+                                fields={
+                                    "id": self.new_id,
+                                    "name": self.action_names,
+                                    "state": "running",
+                                    "created": self.starttime,
+                                    "timestamp": current_time,
+                                },
+                            )
+                        ],
+                        information={self.fqid: ["create action_worker"]},
```

Comment:
please remove. We only want meaningful entries in the history, as introduced in https://github.com/OpenSlides/openslides-backend/pull/1356

Reply:
done